chain/consensus/hierarchical/actors/sca/sca_actor.go

```diff
@@ -781,3 +787,15 @@ func (a SubnetCoordActor) SubmitAtomicExec(rt runtime.Runtime, params *SubmitExe
 	// Return status of the execution
 	return &SubmitOutput{exec.Status}
 }
+
+func recordCrossNetMsgSend(rt runtime.Runtime, msg types.Message, from string, to string, msgType string) {
```

Comment:
This is actor code and runs inside the VM, I don't think you want to track this from the actor code. Even more, this code will disappear soon, as we will remove all LegacyVM code in favor of the FVM, so you'll have to track this from the state and not from its execution.